### PR TITLE
Jakarta MVC 2.1

### PR DESCRIPTION
**Pull request by ivargrimstad**
## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [x] N/A - For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [x] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [x] Updated release record
   - [x] Generated IP Log
   - [x] Email to PMC
   - [x] Start release review by emailing EMO 
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/mvc/jakarta.mvc-api/2.1.0/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      https://download.eclipse.org/ee4j/mvc/staged/2.1/jakarta-mvc-tck-2.1.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/eclipse-ee4j/mvc-api/issues/85
      https://github.com/eclipse-ee4j/mvc-api/issues/86
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

File: mvc/2.1/_index.md [hunk 1]
```diff
@@ -1,23 +1,59 @@
 ---
-title: "Jakarta MVC 2.1 (under development)"
+title: "Jakarta MVC 2.1"
 date: 2021-04-15
 summary: "Release for Jakarta EE 10"
```
Comment (kazumura) [inline mvc/2.1/_index.md:4]: I don't think it relates Jakarta EE 10 release.

Comment (ivargrimstad) [reply mvc/2.1/_index.md:4]: You're right. What we meant by the comment is that it requires EE 10. 
I can change the text to "Release that supports Jakarta EE 10" after the review.

**Comment by Emily-Jiang** [comment]
The [copyright](https://deploy-preview-534--jakartaee-specifications.netlify.app/specifications/mvc/2.1/apidocs/jakarta.mvc/module-summary.html) on the javadoc says 2017, 2020, which should be 2017, 2022.

**Comment by ivargrimstad** [comment]
> The [copyright](https://deploy-preview-534--jakartaee-specifications.netlify.app/specifications/mvc/2.1/apidocs/jakarta.mvc/module-summary.html) on the javadoc says 2017, 2020, which should be 2017, 2022.

We'll fix that in a service release

**Comment by ivargrimstad** [comment]
Hi @keilw I will close this PR now. Please submit a new with the ballot results.

**Pull request merged**